tdrs-backend/tdpservice/search_indexes/migrations/0003_auto_20221208_1750.py

```diff
@@ -0,0 +1,164 @@
+            model_name='t1',
+            old_name='failure_to_comply',
+            new_name='reduc_irp_failure',
+        ),
+        migrations.RenameField(
+            model_name='t1',
+            old_name='reductions_on_receipts',
+            new_name='reduc_length_of_assist',
+        ),
+        migrations.RenameField(
+            model_name='t1',
+            old_name='non_cooperation_cse',
+            new_name='reduc_noncooperation_child_support',
+        ),
+        migrations.RenameField(
+            model_name='t1',
+            old_name='other_sanction',
+            new_name='reduc_other_sanction',
+        ),
+        migrations.RenameField(
+            model_name='t1',
+            old_name='recoupment_prior_ovrpmt',
+            new_name='reduc_prior_overpayment',
+        ),
+        migrations.RenameField(
+            model_name='t1',
+            old_name='sanct_teen_parent',
+            new_name='reduc_teen_not_in_school',
+        ),
+        migrations.RenameField(
+            model_name='t1',
+            old_name='work_req_sanction',
+            new_name='reduc_work_requirements',
+        ),
+        migrations.RenameField(
+            model_name='t1',
+            old_name='sanc_reduction_amount',
+            new_name='reduction_amount',
+        ),
+        migrations.RenameField(
+            model_name='t1',
+            old_name='rpt_month_year',
+            new_name='reporting_month',
+        ),
+        migrations.RenameField(
+            model_name='t1',
+            old_name='cc_amount',
+            new_name='tanf_child_care_amount',
+        ),
+        migrations.RenameField(
+            model_name='t1',
+            old_name='family_exempt_time_limits',
+            new_name='tanf_family_exempt_time_limits',
+        ),
+        migrations.RenameField(
+            model_name='t1',
+            old_name='family_new_child',
+            new_name='tanf_new_child_only_family',
+        ),
+        migrations.RenameField(
+            model_name='t1',
+            old_name='other_total_reductions',
+            new_name='total_reduc_amount',
+        ),
+        migrations.RenameField(
+            model_name='t1',
+            old_name='transp_amount',
+            new_name='transportation_amount',
+        ),
+        migrations.RenameField(
+            model_name='t1',
+            old_name='waiver_evalu_control_grps',
+            new_name='waiver_control_grps',
+        ),
+        migrations.RenameField(
+            model_name='t1',
+            old_name='other_nbr_of_months',
+            new_name='other_nbr_months',
+        ),
+        migrations.RenameField(
+            model_name='t1',
+            old_name='transp_nbr_months',
+            new_name='transport_nbr_months',
+        ),
+        migrations.RemoveField(
+            model_name='t1',
+            name='blank',
+        ),
+        migrations.RemoveField(
+            model_name='t1',
+            name='fips_code',
```

Comment:
Duplicate of `county_fips_code`.

Reply:
@ADPennington @n0remac 

Are county_fips_code and state_fips meaningfully different?

This came up in our sync for #2369

Reply:
yes. county fips is a code for jurisdictions _within_ a state or territory. for the purposes of our system's data reporting, it is 3 digits (with leading zeros in some cases). state fips is 2 digits (with possible leading zeros)    [reference](https://transition.fcc.gov/oet/info/maps/census/fips/fips.txt#:~:text=FIPS%20codes%20are%20numbers%20which,to%20which%20the%20county%20belongs.)

Reply:
Ok, this is a TODO to revert then and resolve. To simplify the migrations, this can be done in-tandem with the planned review of field names.